### ISSUE TITLE
Is the sheepdog still maintaining?Will there be a new version

### ISSUE THREAD

**Issue by fantastic2085** [sheepdog/sheepdog#437]

**Comment by vtolstov** [comment]
No, it does not maintained

**Comment by scream314** [comment]
This is really sad as Sheepdog is the best distributed storage for a lot of workloads/use cases.

**Comment by kloczek** [comment]
Yes please make release even out of currentlyy accumulated commits.
There is already +600 commits since last relese.

**Comment by scream314** [comment]
@kloczek 
I started to sort out the unreleased work, in order to create a new "unofficial" release on a fork (mainly for myself), but unfortunately it is harder and more time consuming than I thought.
As my C knowledge and time is very limited it might take forever.

I also started looking at alternatives, but none of them would fit my needs.
